apps/examples/bluetooth/bt_unit_test_main.c

```diff
@@ -0,0 +1,75 @@
+/****************************************************************************
```

Comment:
If it is not required, just use the same file name with CAPI's unit test such like bt_unit_test.c